application/src/main/java/org/togetherjava/tjbot/commands/basic/SuggestionsUpDownVoter.java

```diff
@@ -0,0 +1,33 @@
+package org.togetherjava.tjbot.commands.basic;
+
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import org.jetbrains.annotations.NotNull;
+import org.togetherjava.tjbot.commands.MessageReceiverAdapter;
+import org.togetherjava.tjbot.config.Config;
+
+import java.util.regex.Pattern;
+
+/**
+ * Listener that receives all sent messages from suggestion channels and reacts with an up- and
+ * down-vote on them to indicate to users that they can vote on the suggestion.
+ */
+public final class SuggestionsUpDownVoter extends MessageReceiverAdapter {
+    /**
+     * Creates a new listener to receive all message sent in suggestion channels.
+     *
+     * @param config the config to use for this
+     */
+    public SuggestionsUpDownVoter(@NotNull Config config) {
+        super(Pattern.compile(config.getSuggestionChannelPattern()));
+    }
+
+    @Override
+    public void onMessageReceived(@NotNull GuildMessageReceivedEvent event) {
+        if (event.getAuthor().isBot() || event.isWebhookMessage()) {
+            return;
+        }
+
+        event.getMessage().addReaction("👍").queue();
+        event.getMessage().addReaction("👎").queue();
```

Comment:
Regardless of the size of this addition, still think we should ^^

Reply:
I would agree if you want me to add a method like `sendReaction(event, "👍")` but even that doesnt really help the code too much.

Reply:
> You want to store up and downvote into constants? I dont think this would serve any purpose other than bloating the code unecessarily.

It's clearer, I don't like that we're looking at an exact emoji, if our system didn't render it it'd be gibberish.
I'd personally rather read `THUMBS_UP_EMOJI`, and I don't see how it bloats the code to code either.

Reply:
Its 2022 and we decided that we should use unicode emojis directly in the code and rely on the fact that all IDEs and websites can render it nowadays. (because of that we also replaced all escaped unicodes in the code base with the actual emojis).

I just feel like it adds no benefit at all to the readability of the code to replace:
```java
addReaction("👍")
```
by
```java
addReaction(THUMBS_UP)
```
in fact, it even distracts the flow cause you have to jump up to the constants to see what it actually is. Or at least "read" the name of the variable concentrated instead of just visually seeing the `👍`.

Reply:
That said, I do not have a very strong opinion here. It doesnt really matter much anyways. So if you insist, I will just change it lol 😆

Reply:
Alright fair enough, @java-coding-prodigy you alright with it not being a constant?

Reply:
I personally think we should put it there, for consistency. I'm fine if we dont use the unicode for it.

Reply:
With the recent changes the constants make more sense (due to being able to give them nice names, i.e. "FALLBACK...").

So I added them, please have a look.